src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import cn from 'classnames';
+
+type Props = {
+  error: string | null;
+  setError: (error: null) => void;
```

Comment:
setError can be of type string or null - setError: (error: string | null) => void;